unit_tests/recovery/ut_database.cpp

```diff
@@ -0,0 +1,363 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#include "ut_database.hpp"
+
+/*******************************************************
+ * ut_database_values_generator
+ *******************************************************/
+
+ut_database_values_generator::ut_database_values_generator (const ut_database_config &a_database_config)
+  : m_database_config (a_database_config)
+  , m_lsa_log_id (10100) // just starts from an arbitrary value
+  , m_gen (m_rd ())
+  , m_duration_in_millis_dist (1, a_database_config.max_duration_in_millis)
+  // *INDENT-OFF*
+  , m_add_or_update_volume_dist ({ut_database_config::ADD_VOLUME_DISCRETE_RATIO,
+                                 ut_database_config::UPDATE_VOLUME_DISCRETE_RATIO })
+  , m_add_or_update_page_dist ({ut_database_config::ADD_PAGE_DISCRETE_RATIO,
+                               ut_database_config::UPDATE_PAGE_DISCRETE_RATIO})
+  // *INDENT-ON*
```

Comment:
Being a cpp file, indent does not run on it, so the comments have no effect.
```suggestion
  , m_add_or_update_volume_dist ({ut_database_config::ADD_VOLUME_DISCRETE_RATIO,
                                 ut_database_config::UPDATE_VOLUME_DISCRETE_RATIO })
  , m_add_or_update_page_dist ({ut_database_config::ADD_PAGE_DISCRETE_RATIO,
                               ut_database_config::UPDATE_PAGE_DISCRETE_RATIO})
```

Reply:
I found that it does. Maybe `astyle` also honors them.